source_code/backend/CS_department_monitor_backend/src/main/java/com/cs5324/monitorbackend/service/EventService.java

```diff
@@ -1,15 +1,67 @@
 package com.cs5324.monitorbackend.service;
 
+import com.cs5324.monitorbackend.entity.Event;
+import com.cs5324.monitorbackend.entity.Notification;
+import com.cs5324.monitorbackend.entity.User;
+import com.cs5324.monitorbackend.entity.enums.ItemStatus;
+import com.cs5324.monitorbackend.exception.EventDoesNotExistException;
 import com.cs5324.monitorbackend.repository.EventRepository;
+import com.cs5324.monitorbackend.repository.UserRepository;
 import jakarta.annotation.Resource;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
 
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Optional;
+import java.util.UUID;
+
 @Slf4j
 @Service
 @RequiredArgsConstructor
 public class EventService{
     @Resource
     private final EventRepository eventRepo;
+
+    @Resource
+    private final UserRepository userRepo;
```

Comment:
We should swap this to utilize the UserService class and not point directly to the repo